src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/CoreNode.cs

```diff
@@ -59,9 +61,9 @@ public class CoreNode
         private string builderWalletPassword;
         private string builderWalletPassphrase;
 
-        public CoreNode(NodeRunner runner, NodeConfigParameters configParameters, string configfile, bool useCookieAuth = false)
+        public CoreNode(NodeRunner runnerParam, NodeConfigParameters configParameters, string configfile, bool useCookieAuth = false)
```

Comment:
why the rename?

Reply:
it collided with the internal item, was not needed finally but I figured its more readable if the class param and method param are different.